update-submodule/README.md

```diff
@@ -8,22 +8,62 @@ feature branch.
 
 ## Parameters
 
+- `repository` — the full name of the target repository: `<owner>/<repo>`.
+  This is the repository that has a submodule to update.
 - `github_token` — the GitHub token of a user with `write` permissions in the 
   target repository. Provide this token as an action secret.
-- `submodule` - the submodule path to update in the target repository; define a 
-  space-separated list to update a few submodules or leave empty to update all.
-- `repository` — the full name of the target repository: `<owner>/<repo>`.
-- `checkout_branch` — the target repository branch to checkout.
-- `feature_branch` — the target repository feature branch.
-- `pr_against_branch` - the target repository branch to open a pull request 
-  against; usually the same as `checkout_branch`.
-- `pr_title` - the title of the pull request; used as the pull request body 
-  as well.
-- `commit_user` - the user for the commit.
-- `commit_user_email` - the user email for the commit.
-- `commit_message` - the message for the commit.
-
-Example workflow:
+- `checkout_branch` — the target repository branch to make a feature branch from;
+  default is `master`.
+- `feature_branch` — the new feature branch in the target repository; 
+  default is `bot/update-submodule`.
+- `submodule` — the submodule path to update in the target repository.
+- `update_to` — update submodule to this git reference; for reliable results,
+  use `SHA1` or tag.
+- `commit_user` — the author of the new commit; default is `TarantoolBot`.
+- `commit_user_email` — the author's email of the new commit;
+  default is `bot@tarantool.io`.
+- `commit_message` — the message for the commit; default is `Update submodule`.
+- `pr_against_branch` — the target repository branch to open a pull request 
+  against; usually the same as `checkout_branch`; default is `master`.
+- `pr_title` — the title of the pull request;
+  default is `[Auto-generated] Update submodule`.
+- `pr_description` — the description (body) of the pull request.
+
+<!-- - `create_pr` — if `true` (default), create a pull request in the target repository. -->
+
+## How it works
+
+First, the action checks out the target repository and makes a `feature_branch`.
+Then is checkouts the `submodule` to `update_to` reference and makes a commit
+with provided name, email and message.
+Finally, it pushes the branch to the target repository:
+
+```mermaid
```

Comment:
Preview: https://github.com/tarantool/actions/tree/nickvolynkin/update-submodule-readme/update-submodule#how-it-works